config/linux-linuxboot.config

```diff
@@ -119,16 +119,16 @@ CONFIG_SCSI_SCAN_ASYNC=y
 CONFIG_SCSI_SPI_ATTRS=y
 CONFIG_SCSI_SAS_LIBSAS=y
 CONFIG_ISCSI_TCP=y
-CONFIG_MEGARAID_NEWGEN=y
-CONFIG_MEGARAID_MM=m
-CONFIG_MEGARAID_MAILBOX=m
-CONFIG_MEGARAID_SAS=m
+# CONFIG_MEGARAID_NEWGEN is not set
+# CONFIG_MEGARAID_MM is not set
+# CONFIG_MEGARAID_MAILBOX is not set
+# CONFIG_MEGARAID_SAS is not set
 CONFIG_SCSI_GDTH=m
 CONFIG_SCSI_ISCI=m
 CONFIG_SCSI_DH=y
 CONFIG_ATA=m
-CONFIG_SATA_AHCI=m
-CONFIG_SATA_AHCI_PLATFORM=m
+CONFIG_SATA_AHCI=y
```

Comment:
The SATA should stay a module - we don't want to spend the time to initialize it and probe the buses on platforms that don't require it (plus it adds to the potential attack surface to initialize them prior to attestation)

Reply:
agree, we shall have I believe a minimal kernel config, but when it is defined as a module it is automatically added to the user land space which makes the system fat. So I think it might be good to make it happens the same way than mlx4 ?